tools/go-cli/Dockerfile

```diff
@@ -34,33 +34,70 @@ RUN mkdir /src/github.com/go-whisk-cli/build
 
 # The BUILDTIME argument is used as the CLI build time property (wsk property get --all)
 # Disabled for now to speed build times
-#ARG BUILDTIME
+ARG LOCAL_BUILD
+ARG LOCAL_OS
+ARG LOCAL_ARCH
+
+RUN echo $LOCAL_ARCH
 
 # Build the Go wsk CLI binaries
-ENV GOOS=windows
 ENV EXEC=wsk.exe
-ENV ARCHS="386 amd64"
-RUN for GOARCH in $ARCHS; do \
+RUN if [ "$LOCAL_BUILD" = "true" ]; then \
+      if [ "$LOCAL_OS" = "windows" ] ; then \
+        export GOOS=$LOCAL_OS && \
+        export ARCHS=$LOCAL_ARCH; \
+      else \
+        unset GOOS && \
+        unset ARCHS; \
+      fi; \
+    else \
+      export GOOS=windows && \
+      export ARCHS="386 amd64"; \
+    fi && \
```

Comment:
I think too that it's easier to read in this shape without much repetition. What would the difference be if it were to have a build script invoked from a single RUN command ?
```
RUN build.sh
```